### ISSUE TITLE
treat README.md as index.md

### ISSUE THREAD

**Issue by Kikobeats** [shuding/nextra-theme-docs#2]
I'm trying to setup nextra for a GitHub documentation and I have to convert all the `README.md` files to be `index.md`

Since README.md is the standard filename used by GitHub to be used as `index.md` in nextra world, why not do that automatically?

**Comment by shuding** [comment]
Yeah that's a good idea, currently it's kinda restricted by Next.js' file-system routing and I don't see an easy way to bypass that. Will keep this issue open for now, thanks!
